eZ/Bundle/EzPublishCoreBundle/DependencyInjection/Compiler/QueryTypePass.php

```diff
@@ -40,7 +42,36 @@ public function process(ContainerBuilder $container)
             }
         }
 
-        // named by convention query types
+        // named by convention in application
+        $applicationRootDir = dirname($container->getParameter('kernel.root_dir'));
+        foreach (glob($applicationRootDir . '/src/QueryType/*QueryType.php') as $queryTypeFilePath) {
```

Comment:
Also the the quality of code in this QueryTypePass class is not acceptable and should be completlly rewriten e.g. code block between lines 47-71 is (almost) copy of https://github.com/ezsystems/ezpublish-kernel/blob/338bb9d8da1fe981dff399b4c462d6cd542062c6/eZ/Bundle/EzPublishCoreBundle/DependencyInjection/Compiler/QueryTypePass.php#L87-L110 
 
Also `QueryTypePassTest` seems to not cover this change.  Is is possible to process thouse changes as separate PR ?